package.json

```diff
@@ -39,6 +42,7 @@
     "nodemon": "^2.0.21",
     "path": "^0.12.7",
     "react": "^18.2.0",
+    "react-custom-scrollbars-2": "^4.5.0",
```

Comment:
I'm not sure if this would go in the category of what Michelle deems a "think that does the thing" for us - might be careful with this here

Reply:
I changed to css for the scroll bar